docs/install/knative-with-operators.md

```diff
@@ -94,9 +120,59 @@ kind: KnativeServing
 metadata:
   name: knative-serving
   namespace: knative-serving
-EOF
+spec:
+  version: ma.mi.pa
+  manifests:
+    - URL: https://github.com/knative/serving/releases/download/v${VERSION}/serving-core.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-hpa.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-post-install-jobs.yaml
+    - URL: https://github.com/knative-sandbox/net-istio/releases/download/v0.17.0/net-istio.yaml
```

Comment:
Should we change `v0.17.0` to `v${VERSION}` too?

Reply:
I would rather keep v0.17.0 for net-istio, because spec.version is for the version of knative.
net-istio is ingress plugin. it could be any supported version.